heron/common/src/python/handler/access/heron.py

```diff
@@ -122,36 +122,64 @@ def get_cluster_topologies(cluster):
   raise tornado.gen.Return((yield fetch_url_as_json(request_url)))
 
```

Comment:
do we need a way to get topologies by cluster and environment too? Should we have an internal method that each of the public ones call?

_get_topologies(cluster, role = None, env = None)